express/tests/middleware/validators/emailIsPresentInSession.test.ts

```diff
@@ -0,0 +1,50 @@
+import {AuthenticationResultType} from "@aws-sdk/client-cognito-identity-provider";
+import {NextFunction, Request, Response} from "express";
+import {Session, SessionData} from "express-session";
+import emailIsPresentInSession from "../../../src/middleware/validators/emailIsPresentInSession/emailIsPresentInSession";
+import {MfaResponse} from "../../../src/services/self-service-services-service";
+
+declare module "express-session" {
+    interface SessionData {
+        emailAddress: string;
+        mobileNumber: string;
+        enteredMobileNumber: string;
+        cognitoSession: string;
+        authenticationResult: AuthenticationResultType;
+        mfaResponse: MfaResponse;
+        updatedField: string;
+        isSignedIn: boolean;
+    }
+}
```

Comment:
We'll need to make this interface public so we don't have to repeat it in tests. Should be enough to move it to its own file 🤞